src/woo_search/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -18,276 +18,331 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: woo_search/accounts/models.py:19
+#: src/woo_search/accounts/models.py:19
 msgid "username"
-msgstr ""
+msgstr "gebruikersnaam"
```

Comment:
You can leave this one empty, the translation shipped with Django itself will pick it up.